packages/salesforcedx-vscode-lwc/src/commands/forceLightningLwcStart.ts

```diff
@@ -153,10 +172,12 @@ export class ForceLightningLwcStartExecutor extends SfdxCommandletExecutor<{}> {
     if (!serverStarted && !cancellationToken.isCancellationRequested) {
       let message = nls.localize('force_lightning_lwc_start_failed');
 
-      // TODO proper exit codes in lwc-dev-server for address in use, auth/org error, etc.
```

Comment:
@sfsholden @mysticflute Is there an existing pattern in lwc-dev-server of process exit code? Are we going to keep using this pattern of generating an "exitCode" from std err?

Reply:
No, we want webruntime to exit with the specific codes when applicable, and also lwc-dev-server itself to exit with the proper code. Then in the extension we simply check for the exit code. I think this solution is temporary until webruntime can be updated to exit with the right code.

Reply:
But if a proper exit code doesn't exist then it should still just be `1`.